### PR TITLE
Melee

### PR DESCRIPTION
**Pull request by Saretzky**

### CODE REVIEW

File: Source/SteveSloth/Private/MyEmuEnemyClass.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /****************************************************************************************
  * Copyright: SteveSloth
  * Name: Elad Saretzky
- * Script: MyEmuEnemyClass.h
+ * Script: MyEmuEnemyClass.cpp
```
Comment (CrazyTobi) [inline Source/SteveSloth/Private/MyEmuEnemyClass.cpp:4]: I can change this in my branch cause I'm working on the emu script, just so there's no conflicts

**Review by CrazyTobi — CHANGES_REQUESTED** [review]
Looks good aside from the one comment

**Comment by Saretzky** [comment]
> Looks good aside from the one comment

done

**Review by CrazyTobi — APPROVED** [review]
Looks good

**Review by AMacLeod99 — APPROVED** [review]

**Pull request merged**